Documentation/Blazorise.Docs/Pages/Docs/Services/ModalProvider/ModalProviderPage.razor

```diff
@@ -22,22 +22,60 @@
             the following example.
         </Paragraph>
         <Paragraph>
-            A <Code>ModalService</Code> will be registered by Blazorise providing you with an API to programatically instantiate modals.
+            A <Code>IModalService</Code> will be registered by Blazorise providing you with an API to programatically instantiate modals.
             Examples are provided further below.
         </Paragraph>
     </DocsPageSectionHeader>
     <DocsPageSectionSource Code="ModalProviderUsageExample" />
 </DocsPageSection>
 
+<DocsPageSection>
+    <DocsPageParagraph>
+        <DocsPageSectionHeader Title="The IModalService">
+            The <Code>IModalService</Code> is an important feature that lets you control <Code Tag>ModalProvider</Code> instantiation.
+            You are provided with various overloads of <Code>Show</Code> to instantiate the <Code Tag>Modal</Code> to your liking.
+            This will return a <Code>ModalInstance</Code> that represents the <Code Tag>Modal</Code> you've instantiated with all of its configuration.
+            You may also use the provided <Code>Hide</Code> to close Modals.
+        </DocsPageSectionHeader>
+    </DocsPageParagraph>
+</DocsPageSection>
+
 <DocsPageSection>
     <DocsPageSectionHeader Title="Options">
         You may provide options to <Code Tag>ModalProvider</Code> globally by setting them on the <Code Tag>ModalProvider</Code> component itself.
+    </DocsPageSectionHeader>
+    <DocsPageSectionSource Code="ModalProviderOptionsExample" />
+    <DocsPageSectionHeader>
         Optionally you can override these when instantiating a modal through the <Code>ModalService</Code> usage.
     </DocsPageSectionHeader>
-    <DocsPageSectionSource Code="ModalProviderOptionsUsageExample" />
     <DocsPageSectionSource Code="ModalServiceOptionsExample" />
 </DocsPageSection>
 
+<DocsPageSection>
+    <DocsPageParagraph>
```

Comment:
DocsPageParagraph should be on a page level.

Sections hierarchy is:

- DocsPageSection
  - DocsPageSectionHeader
  - DocsPageSectionContent
  - DocsPageSectionSource

where regular Paragraph can be used inside of a DocsPageSectionHeader or DocsPageSectionContent